docs/pages/application-access/getting-started.mdx

```diff
@@ -9,85 +9,92 @@ videoBanner: 5Uwhp3IQMHY
+- A Docker installation, which we will use to launch Grafana in a container. Alternatively, if you have another web application you'd like to protect with Application Access, you can use that instead.
+- A host where you will run the Teleport Application Service.
+
+We will assume your Teleport cluster is accessible at `teleport.example.com` and `*.teleport.example.com`. You can substitute the address of your Teleport Proxy Service. (For Teleport Cloud customers, this will be similar to `mytenant.teleport.sh`.)
+
+<Admonition type="tip" title="Not yet a Teleport user?">
+If you have not yet deployed the Auth Service and Proxy Service, you should follow one of our [getting started guides](../getting-started.mdx).
+</Admonition>
 
 ## Step 1/3. Start Grafana
 
 We've picked Grafana for this tutorial since it's very easy to run with zero
 configuration required. If you have another web application you'd like to
 expose, skip over to **Step 2**.
 
-Grafana can be launched in a [Docker container](https://grafana.com/docs/grafana/latest/installation/docker/)
-with a single command:
+Grafana can be launched in a Docker container with a single command:
 
 ```code
 $ docker run -d -p 3000:3000 grafana/grafana
 ```
 
 ## Step 2/3. Install and configure Teleport
 
-Download the latest version of Teleport for your platform from our
-[downloads page](https://goteleport.com/teleport/download).
-
-Teleport requires a valid TLS certificate to operate and can fetch one automatically
-using Let's Encrypt [ACME](https://letsencrypt.org/how-it-works/) protocol.
-
-We will assume that you have configured DNS records for `teleport.example.com`
-and `*.teleport.example.com` to point to the Teleport node.
-
-(!docs/pages/includes/permission-warning.mdx!)
-
-Let's generate a Teleport config with ACME enabled:
+### Generate a token
+A join token is required to authorize a Teleport Application Service agent to
+join the cluster. Generate a short-lived join token and save it, for example,
+in `/tmp/token` on your Teleport Application Service host:
 
 ```code
-$ sudo teleport configure --cluster-name=teleport.example.com --acme --acme-email=alice@example.com -o file
+$ tctl tokens add \
+    --type=app \
+    --app-name=grafana \
+    --app-uri=http://localhost:3000
 ```
 
-<Admonition
-  type="note"
-  title="Web Proxy Port"
->
-  Teleport uses [TLS-ALPN-01](https://letsencrypt.org/docs/challenge-types/#tls-alpn-01)
-  ACME challenge to validate certificate requests which only works on port `443`. Make sure your Teleport proxy is accessible on port `443` when using ACME for certificate management.
-</Admonition>
+### Start Teleport
+On the host where you will run the Teleport Application Service, download the latest version of Teleport for your platform from our
+[downloads page](https://goteleport.com/teleport/download).
 
 Now start Teleport and point it to the application endpoint:
 
 ```code
 $ sudo teleport start \
```

Comment:
Should we prefer `teleport app start` instead of `teleport start --roles=app`?

- `--app-uri` becomes `--uri`
- `--app-name` becomes `--name`
- `--roles` can be removed
- `--token` and `--auth-server` remain unchanged

Reply:
If we do prefer `teleport app start` here, then these docs become inconsistent with the manual instructions built into Teleport Cloud > Applications > Add Application > Manually, where it says:

`teleport start --roles=app --app-name=[example-app] --app-uri=http://localhost/ --token=[generated-join-token] --auth-server=example.cloud.gravitational.io:443` 

As an aside, I found Teleport Cloud's automatic method of adding app access (via the auto-installer script) super easy.    I bet most people will use this.

Reply:
@zmb3 What do you think? Shall we open a PR against `gravitational/webapps` to change the [help text](https://github.com/gravitational/webapps/blob/2bf39459e741942d305ffe2d179833dc3b5934ac/packages/teleport/src/Apps/AddApp/Manually/Manually.tsx#L109) and we can merge these two changes at the same time?